unicodetools/org/unicode/propstest/CheckEmojiProps.java

```diff
@@ -0,0 +1,25 @@
+package org.unicode.propstest;
+
+import org.unicode.props.IndexUnicodeProperties;
+import org.unicode.props.UcdProperty;
+import org.unicode.props.UcdPropertyValues.Binary;
+
+import com.google.common.collect.Sets;
+import com.ibm.icu.text.UnicodeSet;
+
+public class CheckEmojiProps {
+    static IndexUnicodeProperties IUP = IndexUnicodeProperties.make();
+
+    static UnicodeSet EMOJI = IUP.loadEnum(UcdProperty.Emoji, Binary.class).getSet(Binary.Yes);
+    static UnicodeSet BASIC_EMOJI = IUP.loadEnum(UcdProperty.Basic_Emoji, Binary.class).getSet(Binary.Yes);
+    static UnicodeSet EMOJI_COMPONENT = IUP.loadEnum(UcdProperty.Emoji_Component, Binary.class).getSet(Binary.Yes);
+    static UnicodeSet ZWJ_EMOJI = IUP.loadEnum(UcdProperty.RGI_Emoji_Zwj_Sequence, Binary.class).getSet(Binary.Yes);
+    public static void main(String[] arg) {
+	UnicodeSet b_c = new UnicodeSet( EMOJI_COMPONENT).retainAll(BASIC_EMOJI);
```

Comment:
The logic is right, but the names of the variables b_c and bAndc are the opposite of what they contain, assuming that `_` stands for subtraction, and "And" for intersection.

Reply:
ping

Reply:
I just suggested a change on the current file version.

Reply:
applied